### ISSUE TITLE
when will unified reward be released?

### ISSUE THREAD

**Issue by yxchng** [Tencent-Hunyuan/MixGRPO#2]
is current version different from the paper since unified reward is not yet released?

**Comment by tulvgengenr** [comment]
Thank you very much for your interest in our work. In fact, we have provided the code for the unified reward client (see https://github.com/Tencent-Hunyuan/MixGRPO/blob/main/fastvideo/models/reward_model/unified_reward.py), while retaining the code for the unified reward section in the training script. 

If you want to quickly reproduce it, you can build your own server based on the client interface definition.

Good luck!

**Comment by tulvgengenr** [comment]
@yxchng Yes, we use the UnifiedReward-qwen-7b
